### PR TITLE
game8.jp - PC 

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 https://game8.jp/battlecats/207051

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -1156,7 +1155,10 @@ how-to-inc.com##.adblocks
 ||game8.jp/assets/*/ads/
 ||game8.jp/assets/pc/jack_ad/
 game8.jp###g8-ad-placement-pc_side_head
+game8.jp,nurumayu.net###js-popup
+game8.jp##.ad-skyflag-wrapper
```
Comment (Alex-302) [inline JapaneseFilter/sections/specific.txt:1159]: not found

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:1159]: `game8.jp,nurumayu.net###js-popup`  is added in this issue https://github.com/AdguardTeam/AdguardFilters/issues/123677.  I can not reproduce.
Should I remove this rule?

`game8.jp##.ad-skyflag-wrapper`is added in this issue  https://github.com/AdguardTeam/AdguardFilters/issues/124320. I can reproduc on mobile. 
Should I move this rule to mobile filter?

**Comment by SKEIDs** [comment]
> Please don't change the rules order.

sorry

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply JapaneseFilter/sections/specific.txt:1159]: Removed game8.jp from `###js-popup`

Comment (Alex-302) [reply JapaneseFilter/sections/specific.txt:1159]: > Should I move this rule to mobile filter?

Moved.

Comment (Alex-302) [reply JapaneseFilter/sections/specific.txt:1159]: Please update PR.

Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt]: ok,done

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
thanks

**Pull request merged**